legend-engine-xts-dataquality/legend-engine-xt-dataquality-pure-test/src/main/resources/core_dataquality_test/dataquality_test.pure

```diff
@@ -170,4 +170,38 @@ function <<test.Test>> meta::external::dataquality::tests::testMetadata():Boolea
     
     //TODO assert on structure.
     assert($metadata->isNotEmpty());
+}
+
+function <<test.Test>> meta::external::dataquality::tests::testLambdaGeneration_relationValidation():Boolean[1]
+{
+  doRelationTest('###DataQualityValidation' +
```

Comment:
can we simplify to reduce the boilerplate being passes in similar to the non-relation tests?

Reply:
The code under test is the only input variable here which makes more sense to me than constructing the code under test using individual params